keyboards/jiran/rules.mk

```diff
@@ -0,0 +1,39 @@
+MCU = atmega32u4
+F_CPU = 16000000
+ARCH = AVR8
+F_USB = $(F_CPU)
+
+# Bootloader
+#     This definition is optional, and if your keyboard supports multiple bootloaders of
+#     different sizes, comment this out, and the correct address will be loaded
+#     automatically (+60). See bootloader.mk for all options.
+ifeq ($(strip $(KEYBOARD)), jiran/rev1)
+    BOOTLOADER = dfu
```

Comment:
Set this in the rev-specific rules.mk, not here. And `dfu` is not a valid bootloader type.